### PR TITLE
Added master league and improved GUI slightly

### PR DESCRIPTION
**Pull request by kimjudyh**
- added master league power up costs analysis
- added option to view said analysis in GUI
- cleaned up GUI code a bit, ex. removed extraneous comments and blank lines

### CODE REVIEW

File: pogo_gui.py [hunk 1]
```diff
@@ -6,19 +6,20 @@
 from multi_poke_v1 import *
 
 # can remove this function, i think
+'''
```
Comment (jeffsetter) [inline pogo_gui.py]: delete

File: pogo_gui.py [hunk 2]
```diff
@@ -6,19 +6,20 @@
...
 def get_option_states():
     ultra_state = show_ultra_league.get()
     return ultra_state
+    '''
 
 def db_search(search_value):
     # search for pokemon names that match search string
     # make results global so can be accessed in other functions
-    global search_db
+    #global search_db
```
Comment (jeffsetter) [inline pogo_gui.py]: delete

File: multi_poke_v1.py [hunk 3]
```diff
@@ -55,12 +56,14 @@ def evo_pokemon_input(e_poke=None):
 
 
 # get external status on whether to display Ultra League analysis
-def get_display_option(state):
+def get_display_option(state1, state2):
```
[the remark this answers is not in the document]
Comment (kimjudyh) [reply multi_poke_v1.py]: done

File: pogo_gui.py (hunk 1 above)
Comment (kimjudyh) [reply pogo_gui.py]: done

File: pogo_gui.py (hunk 2 above)
Comment (kimjudyh) [reply pogo_gui.py]: done

**Pull request merged**